app/models/configuration_script.rb

```diff
@@ -1,3 +1,10 @@
 class ConfigurationScript < ActiveRecord::Base
   belongs_to :configuration_manager, :class_name => 'ManageIQ::Providers::ConfigurationManager'
+
+  def run
```

Comment:
@Fryguy - Would you rather see launch? - We decided on run because it had a general enough name.